### ISSUE TITLE
Remove pdfs from the repo

### ISSUE THREAD

**Issue by cxkoda** [posquit0/Awesome-CV#317]
To keep the repository tidy I suggest removing the pdfs entirely from the repository.  
Since they are obtained directly from building they add no additional information and should hence not be part of the repository. Instead they pdfs should be obtained from the ci-pipeline.

What do you think?

**Comment by urbanhusky** [comment]
I think that the pdfs are a huge benefit because they give a proper preview. 
The examples might also not build right out of the box (on Windows).

**Comment by cxkoda** [comment]
I agree the pdfs are very nice for the preview. But they bloat the whole repository, especially since the have to be rebuilt and updated after every change.

This can be avoided if we build the pdfs through the ci-pipeline and set links in the readme, accordingly. Check out my pull request, or the master branch of my fork if you are interested how this would look like.

**Comment by robglas** [comment]
That makes a ton of sense to me. **If** said examples are still readily viewable (since that's what draws new people in / gets them interested).

**Comment by cxkoda** [comment]
In the readme of my [fork](https://github.com/cxkoda/Awesome-CV) I set the preview links to point to the respective circle-artifacts (e.g. https://circleci.com/api/v1.1/project/github/cxkoda/Awesome-CV/latest/artifacts/0/~/results/examples/resume.pdf?branch=master&filter=successful), which are rebuild by the ci-pipeline after every push.
